examples/datasets/download_dataset.py

```diff
@@ -1,22 +1,35 @@
-"""Script to download benchmark dataset(s)"""
+"""
+Script to download benchmark dataset(s)
+
+By default, this script downloads the 'mipnerf360' dataset. 
+You can specify a different dataset to download using the --dataset option. 
+If you want to download all available datasets, you can set the --dataset option to 'all'.
+"""
 
 import os
 import subprocess
 from dataclasses import dataclass
 from pathlib import Path
-from typing import Literal
-
+from typing import Literal, List
 import tyro
 
 # dataset names
-dataset_names = Literal["mipnerf360"]
+dataset_names = Literal["mipnerf360", "mipnerf360_extra", "tandt", "deepblending", "all"]
 
 # dataset urls
-urls = {"mipnerf360": "http://storage.googleapis.com/gresearch/refraw360/360_v2.zip"}
+urls = {
+    "mipnerf360": "http://storage.googleapis.com/gresearch/refraw360/360_v2.zip",
+    "mipnerf360_extra": "https://storage.googleapis.com/gresearch/refraw360/360_extra_scenes.zip",
+    "tandt_db": "https://repo-sam.inria.fr/fungraph/3d-gaussian-splatting/datasets/input/tandt_db.zip"
+}
```

Comment:
Seems the url for `deepblending` is missing?

Reply:
The link for "tandt_db" is from the original 3DGS project and contains the relevant scenes from t&t and deepblending